presto-docs/src/main/sphinx/sql/show-role-grants.rst

```diff
@@ -0,0 +1,17 @@
+================
+SHOW ROLE GRANTS
+================
+
+Synopsis
+--------
+
+.. code-block:: none
+
+    SHOW ROLE GRANTS [[FOR] [user | USER user | ROLE role]] [ FROM catalog ]
```

Comment:
If we want to support `SHOW ROLE GRANTS FOR`, make sure it could be executed only by admin.
`SHOW ROLE GRANTS` should be allowed to execute by everyone. But it should display only it's granted roles.